### PR TITLE
Feature/object management list fields tab

### PR DESCRIPTION
**Pull request by Kevin-Snijders**
The backend was still in progress so I had multiple tabs in my branch. The new tabs are in Carbon style and work with the new backend. The only thing that doesn't work yet is the reorder button in the lists which should be discussed with Niek & Klaas I guess.

The story includes #43763, #43762, #43761, #50726, #43755, #43754

### CODE REVIEW

File: projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+    this.selectedFieldTypeItemIndex$.next(0);
+  }
+
+  private mapFormValuesToColumn(formValue: any): SearchField {
+    return {
+      key: formValue.key,
+      title: formValue.title || '',
+      path: formValue.path,
+      dataType: formValue.dataType.key,
+      fieldType: formValue.fieldType.key
```
Comment (Klaas-Ritense) [inline projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts:425]: This gives an error: 'Missing matchType'

Butnew searchFields don't have a matchType

Comment (Kevin-Snijders) [reply projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts:425]: You need to update the config package.

Comment (Klaas-Ritense) [reply projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts:425]: I did a rebuild locally and it worked. But now I see that the build failed with the same error:
```
error TS2741: Property 'matchType' is missing in type '{ key: any; title: any; path: any; dataType: any; fieldType: any; }' but required in type 'SearchField'.
```

Comment (Kevin-Snijders) [reply projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts:425]: I will take a look at it.

Comment (Kevin-Snijders) [reply projects/valtimo/object-management/src/lib/components/object-management-detail-container/tabs/object-management-list-search-fields/object-management-list-search-fields.component.ts:425]: I made a new commit which fixes this issue.

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**